dao/definitions_dao.go

```diff
@@ -79,14 +80,14 @@ func GetTaskDefinitionById(definitionId uuid.UUID) (models.TaskDefinition, error
 	return task_def, result.Error
 }
 
-func UpdateDefinition(taskDefinition models.TaskDefinition) {
+func UpdateDefinition(taskDefinition *models.TaskDefinition) {
 	result := initializers.GetTaskDefinitionsTable().Where(
 		"id = ?",
 		taskDefinition.ID,
 	).Where(
 		"deleted = ?", // Can't update deleted definitions
 		false,
-	).Updates(
+	).Save(
```

Comment:
hmm why this change?

Reply:
Updates doesn't update fields with 0 value.
So the `enabled=false` field wasn't updated. This meant that a one-time task would run again and again each time the poller runs